crates/gpui/src/window.rs

```diff
@@ -3610,7 +3610,11 @@ impl<'a> WindowContext<'a> {
         self.on_next_frame(|cx| {
             if let Some(mut input_handler) = cx.window.platform_window.take_input_handler() {
                 if let Some(bounds) = input_handler.selected_bounds(cx) {
-                    cx.window.platform_window.update_ime_position(bounds);
+                    let scaled_bounds = Bounds {
+                        origin: bounds.origin.map(|coord| coord * cx.window.scale_factor),
+                        size: bounds.size.map(|dim| dim * cx.window.scale_factor)
+                    };
```

Comment:
Can you update the code to use the existing methods and types for scaling the pixels?

Like this:

```suggestion
                    let scaled_bounds = bounds.scale(cx.scale_factor());
```

The result is a `Bounds<ScaledPixels>`, which means the type for `update_ime_position` needs to change, which I think is fine. What do you think?